translations/rtd2_perks.phrases.txt

```diff
@@ -820,7 +820,7 @@
 		"en"	"Spy is gonna get you! Or will he?"
 		"pl"	""
 		"chi"	""
-		"ru"	""
+		"ru"	"Шпион доберется до вас! Но... кто он?"
```

Comment:
I see this roughly translates to `The spy will get you! But... who is he?`. The second part about who the spy is should be something else.

This perk makes you hear spy noises and see fake disguise smoke effects around you. So it's not a question of who the spy is but rather if he is real.

Reply:
> I see this roughly translates to `The spy will get you! But... who is he?`. The second part about who the spy is should be something else.
> 
> This perk makes you hear spy noises and see fake disguise smoke effects around you. So it's not a question of who the spy is but rather if he is real.

ah, thank you for the info! i didn't know about that new effect so should i translate second part as ``But... where is he?``

Reply:
or ``Hm... Is my mind playing tricks on me?``

Reply:
> or `Hm... Is my mind playing tricks on me?`

Yeah that sound good!

Reply:
Hey, I'm actually hoping to release 2.4.0 soon. Any chance you could throw it in now?